instrumentation/elastic-opentelemetry-instrumentation-openai/src/opentelemetry/instrumentation/openai/__init__.py

```diff
@@ -25,15 +25,15 @@
     OTEL_INSTRUMENTATION_GENAI_CAPTURE_MESSAGE_CONTENT,
 )
 from opentelemetry.instrumentation.openai.helpers import (
+    _get_embeddings_span_attributes_from_response,
     _get_embeddings_span_attributes_from_wrapper,
     _get_event_attributes,
+    _get_span_attributes_from_response,
```

Comment:
I would probably remove "span" from these methods/variables, just attributes, though the "full attributes of an operation" do tend to be the same as the ones used for the span.

Reply:
Good point, I'll rename them since we are already touching every caller